Xamarin.Forms.Platform.Android/FastRenderers/LabelRenderer.cs

```diff
@@ -292,9 +292,9 @@ void UpdateFont()
 			}
 		}
 
-		void UpdateTextDecorations()
+		void UpdateTextDecorations(bool isNewElement)
```

Comment:
I wonder if instead of adding this property we just do an if check inside *OnElementChanged* to not call UpdateTextDecorations since that's really the only scenario this bool is accounting for

Reply:
Yeah, we could just check for !isSet in that call instead of passing the bool, I can change it to work that way